### PR TITLE
feat: autofocus as a prop

### PR DESCRIPTION
**Pull request by drdelambre**
this was brought up by @nathanielc but i can't find the issue

whenever you have a notebook that is just a wall of flux panels, if you come back to it... each of the flux panels are autofocused, meaning you are looking at one on them, but anything you type is applied to all of them. real bummer when you `cmd-a` + `cmd-v` that sweet updated flux function a friend sent you, just to have it wipe out ALL of your panels.

this turns off autofocus on notebooks until we can figure out a way to say "autofocus, but only on the last one"

### CODE REVIEW

File: src/tasks/containers/TaskEditPage.tsx [hunk 1]
```diff
@@ -91,6 +91,7 @@ class TaskEditPage extends PureComponent<Props> {
                 <FluxMonacoEditor
                   script={currentScript}
                   onChangeScript={this.handleChangeScript}
+                  autofocus
```
Comment (genehynson) [inline src/tasks/containers/TaskEditPage.tsx:94]: isn't this turning _on_ autofocus?

**Review by genehynson — APPROVED** [review]
lgtm except it looks like you're turning _on_ autofocus, right? Description says you're turning it off but maybe I'm missing something

**Comment by drdelambre** [comment]
lol it is kinda backwards. i'm wrapping the logic that used to default to being on behind a prop that defaults to off, then turning it on for the components that wanted it to be on, and omitting the addition for the ones that need it to be off.

**Comment by genehynson** [comment]
> lol it is kinda backwards. i'm wrapping the logic that used to default to being on behind a prop that defaults to off, then turning it on for the components that wanted it to be on, and omitting the addition for the ones that need it to be off.

ohhhhhh got it

**Comment by drdelambre** [comment]
all of this is because i like declarative props that describe the intended outcome more than props that pose themselves as a question like `shouldDisableAutofocus`

**Comment by ui-ci-bot-influx** [comment]
Metric | Score | Level
------------ | ------------- | ------------- 
performance | 57 | :red_circle:
accessibility | 97 | :green_circle:
seo | 90 | :green_circle:
pwa | null | :hankey:
best-practices | 93 | :green_circle:


[Compare your scores with master branch](https://influxdb.aws.influxdata.io/orgs/844910ece80be8bc/dashboards/073c4b95f1089000?lower=now%28%29%20-%202d&vars%5Blighthouse_score_branch%5D=3195)

File: src/shared/selectors/flags.ts [hunk 2]
```diff
@@ -52,7 +51,6 @@ export const OSS_FLAGS = {
 
 export const CLOUD_FLAGS = {
   cloudBilling: CLOUD_BILLING_VISIBLE, // should be visible in dev and acceptance, but not in cloud
-  cursorAtEOF: false,
```
Comment (asalem1) [inline src/shared/selectors/flags.ts:55]: Pretty sure you checked that this exists in config cat too?

**Pull request merged**